src/applications/vaos/services/appointment/transformers.js

```diff
@@ -18,22 +18,14 @@ export function getAppointmentType(
   // TODO: Update APPOINTMENT_TYPES enum to match API response values.
   const isCerner = appt?.id?.startsWith('CERN');
 
-  // In an upcoming iteration, we will be update the FE source of truth for VA requests as well
-  // eslint-disable-next-line sonarjs/no-collapsible-if
   if (useFeSourceOfTruthVA) {
-    if (isCerner && appt?.type === 'VA') {
-      return APPOINTMENT_TYPES.vaAppointment;
-    }
-  } else {
-    // In an upcoming iteration, we will be update the FE source of truth for VA requests as well
-    // eslint-disable-next-line no-lonely-if
-    if (isCerner && !isEmpty(appt?.end)) {
+    if (appt?.type === 'VA') {
```

Comment:
Basically refactored the if statement back to the original if statement and added the new if statement with the feature flags separately. Hopefully, the refactor is easier to read and understand.